test/expected/create_hypertable.out

```diff
@@ -974,3 +974,28 @@ ERROR:  insert or update on table "_hyper_21_21_chunk" violates foreign key cons
 DROP TABLE regular cascade;
 NOTICE:  drop cascades to 2 other objects
 DROP TABLE timescale cascade;
+-- github issue 4872
+create table some_hypertable (
+  time timestamp with time zone,
+  some_column bigint
+);
+select create_hypertable('some_hypertable', 'time');
+NOTICE:  adding not-null constraint to column "time"
+       create_hypertable       
+-------------------------------
+ (22,public,some_hypertable,t)
+(1 row)
+
+insert into some_hypertable (time, some_column) values ('2023-02-12 00:00:00+02:40'::timestamp with time zone, 1);
+insert into some_hypertable (time, some_column) values ('2023-02-12 00:00:10+02:40'::timestamp with time zone, 2);
+insert into some_hypertable (time, some_column) values ('2023-02-12 00:00:20+02:40'::timestamp with time zone, 3);
+-- should not fail with error
```

Comment:
Please give some context to why this failed with an error before and what this test is about. A future developer might remove this check because it looks innocent enough and it is not clear that this triggers a TID range scan.

Reply:
Fixed.